pkg/util/dsl/template.go

```diff
@@ -41,7 +41,7 @@ func TemplateToYamlDSL(template v1.Template, id string) ([]byte, error) {
 		jsondata["id"] = id
 	}
 	templateWrapper := make(map[string]interface{}, 2)
-	templateWrapper["template"] = jsondata
```

Comment:
I think the usage of template is still good. However, instead of being an array (therefore a list in yaml) it should be a single parameter of the spec.

Reply:
I removed the array, but that bring a conversion error in camel yaml dsl. To proceed with having `main` in a good state, can we merge as it is, and we can review this array later  ?

Reply:
+1 for me, I'll wait for @squakez advice

Reply:
Thanks. I had a deeper look and it seems that `template` is no longer there. So we should we good with `route-template`.